screens/ProfileScreen/Navigation/ProfileScreensNavigation.tsx

```diff
@@ -0,0 +1,42 @@
+import { StackScreenProps } from "@react-navigation/stack"
+import { ProfileScreenProps } from "../ProfileScreen"
+import { StackNavigatorType } from "@components/Navigation";
+import { SettingsScreen } from "@screens/SettingsScreen/SettingsScreen";
+import { ProfileScreenNavWrapper } from "./ProfileScreenNavWrapper";
+import { EditProfileScreenNavWrapper } from "./EditProfileScreenNavWrapper";
+
+
+export type ProfileScreensParamsList = {
+  "Profile Screen": ProfileScreenProps
+  "Edit Profile Screen": ProfileScreenProps
```

Comment:
Also, reminder that we only want an id for whoever's profile it is as the route params, so we can get deep linking. For the case of the current user's profile tab, we can just have a special route that leads to that screen which doesn't need an id prop since we know it's the current user.